dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/StreamingResponse.cs

```diff
@@ -41,6 +41,11 @@ public class StreamingResponse
         /// </summary>
         public bool? EnableFeedbackLoop { get; set; } = false;
 
+        /// <summary>
+        /// Represents the type of feedback loop. Set to "default" by default. It can be set to one of "default" or "custom".
+        /// </summary>
+        public string FeedbackLoopType { get; set; } = "default";
```

Comment:
should we make this an enum?

Reply:
I can do that

Reply:
upon offline conversation, we've decided to keep it as a string so that it is flexible to citations api changes.